### PR TITLE
Better CN callouts for TEA pre-observation

### PR DESCRIPTION
**Pull request by miaonima**
Make buff names easier to recognize and consistent with TEA Beta Instructions Callout, and provide instructions for ORANGE BAIT and PURPLE BAIT (as in other translations). Confirmed this is the popular strategy (if not the only viable strategy) for this set of mechanics on CN servers.

I saw many raiders in the CN TEA discussion group would like to disable these 4 triggers because of the confusing official names. The buffs are more easily recognized by the color rather than their official names.

### CODE REVIEW

**Comment by Trim21** [comment]
Not a good idea... Maybe `大光` `大暗` is even better

**Comment by miaonima** [comment]
> Not a good idea... Maybe `大光` `大暗` is even better

I agree. Will close this and revise.

File: ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import Conditions from '../../../../../resources/conditions.ts';
+import Conditions from '../../../../../resources/conditions.js';
 import NetRegexes from '../../../../../resources/netregexes.js';
-import Outputs from '../../../../../resources/outputs.ts';
+import Outputs from '../../../../../resources/outputs.js';
```
Comment (Trim21) [inline ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js:3]: why making this change？

**Comment by miaonima** [comment]
Sorry for the mess! I'm quite new to git. Please let me know if this is correct now.

**Comment by quisquous** [comment]
Seems good to me to match what other languages are doing, especially if other people don't like it.  Thanks for the feedback!

@Trim21 does this look good to you?

(Also, sorry that git is confusing, especially when you're new to it.  Thanks for making it work!  I am happy to help in the future if git behaves badly and you have questions or need something fixed.)

**Comment by Trim21** [comment]
@quisquous LGTM

**Pull request merged**